user.go

```diff
@@ -0,0 +1,23 @@
+package mockoidc
+
+// User defines the expected user values
+type User struct {
+	Email             string   `json: email`
```

Comment:
Not sure we need struct tags for this (maybe for userinfo implementation later). But you'll need to tweak the syntax a bit:
```suggestion
	Email             string   `json:"email,omitempty"`
```